Source: @pullmoll
Upstream: no
Reason: qt5-5.15.0 requires explicit include for QPainterPath

--- src/gui/AutomationPatternView.cpp	2018-11-03 02:43:42.000000000 +0100
+++ src/gui/AutomationPatternView.cpp	2020-06-30 20:54:11.520449601 +0200
@@ -25,6 +25,7 @@
 
 #include <QMouseEvent>
 #include <QPainter>
+#include <QPainterPath>
 #include <QMenu>
 
 #include "AutomationEditor.h"
--- src/gui/LmmsStyle.cpp	2018-11-03 02:43:42.000000000 +0100
+++ src/gui/LmmsStyle.cpp	2020-06-30 22:59:34.411837701 +0200
@@ -28,6 +28,7 @@
 #include <QApplication>
 #include <QFrame>
 #include <QPainter>
+#include <QPainterPath>
 #include <QStyleFactory>
 #include <QStyleOption>
 
--- src/gui/editors/AutomationEditor.cpp	2018-11-03 02:43:42.000000000 +0100
+++ src/gui/editors/AutomationEditor.cpp	2020-06-30 23:03:53.262851055 +0200
@@ -35,6 +35,7 @@
 #include <QLayout>
 #include <QMdiArea>
 #include <QPainter>
+#include <QPainterPath>
 #include <QScrollBar>
 #include <QStyleOption>
 #include <QToolTip>
--- plugins/Eq/EqSpectrumView.h	2018-11-03 02:43:42.000000000 +0100
+++ plugins/Eq/EqSpectrumView.h	2020-06-30 23:07:48.389863185 +0200
@@ -24,6 +24,7 @@
 #define EQSPECTRUMVIEW_H
 
 #include <QPainter>
+#include <QPainterPath>
 #include <QWidget>
 
 #include "fft_helpers.h"
--- plugins/Eq/EqCurve.h	2018-11-03 02:43:42.000000000 +0100
+++ plugins/Eq/EqCurve.h	2020-06-30 23:08:21.792864908 +0200
@@ -27,6 +27,7 @@
 
 #include <QGraphicsItem>
 #include <QPainter>
+#include <QPainterPath>
 #include <QGraphicsSceneWheelEvent>
 #include "lmms_math.h"
 #include "AutomatableModelView.h"
